README.md

```diff
@@ -67,6 +67,7 @@ SensorHCSR04             | 1     | USE_HCSR04         | HC-SR04 sensor, return t
 SensorMCP9808            | 1     | USE_MCP9808        | MCP9808 sensor, measure the temperature through the attached module                               | https://github.com/adafruit/Adafruit_MCP9808_Library
 SensorMQ                 | 1     | USE_MQ             | MQ sensor, return ppm of the target gas                                                           | -
 SensorMHZ19              | 1     | USE_MHZ19          | MH-Z19 CO2 sensor via UART (SoftwareSerial, default on pins 6(Rx) and 7(Tx)                       | -
+SensorSDS011             | 2     | USE_SDS011         | SDS011 air quality sensor, return concentrations of 2.5 and 10 micrometer particles.              | https://github.com/ricki-z/SDS011
```

Comment:
May you please add the same into the same table of NodeManager.ino? Also if possibile, push it to the end after SensorNeopixel so the user will better understand what is new compared to the old version